### PR TITLE
Add information for using JNA

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1499.surge.sh

Updated pages:
- [Default file locations](https://neo4j-docs-operations-1499.surge.sh/operations-manual/5/configuration/file-locations)
- [System requirements](https://neo4j-docs-operations-1499.surge.sh/operations-manual/5/installation/requirements)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1499.surge.sh

Updated pages:
- [Default file locations](https://neo4j-docs-operations-1499.surge.sh/operations-manual/5/configuration/file-locations)
- [System requirements](https://neo4j-docs-operations-1499.surge.sh/operations-manual/5/installation/requirements)

File: modules/ROOT/pages/configuration/file-locations.adoc [hunk 1]
```diff
@@ -15,7 +15,9 @@ Instructions provided for Neo4j Desktop are applicable across all operating syst
 If  _tmp_ is set to `noexec`, it is recommended to set `server.jvm.additional=-Djava.io.tmpdir=/home/neo4j` in _conf/neo4j.conf_.
 Additionally, replace _/home/neo4j_ with a path that has `exec` permissions.
 
-For _/bin/cypher-shell_, set this via an environment variable: `export JAVA_OPTS=-Djava.io.tmpdir=/home/neo4j` and replace `/home/neo4j` with a path that has `exec` permissions.
+Similarly, if are using the Java Native Access (JNA) library, set `server.jvm.additional=-Djna.tmpdir=/tmp` in _conf/neo4j.conf_, and replace _/tmp_ with a path that has `exec` permissions.
```
Comment (docrozza) [inline modules/ROOT/pages/configuration/file-locations.adoc:18]: not sure about the `if` here - the server itself is using JNA to provide native access to system calls so we use it on linux machines

maybe something like
> Neo4j's uses of the Java Native Access (JNA) library also requires _tmp_ to be set to `noexec` so set `server.jvm.additional=-Djna.tmpdir=/tmp` in _conf/neo4j.conf_, and replace _/tmp_ with a path that has `exec` permissions.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1499.surge.sh

Updated pages:
- [Default file locations](https://neo4j-docs-operations-1499.surge.sh/operations-manual/5/configuration/file-locations)
- [System requirements](https://neo4j-docs-operations-1499.surge.sh/operations-manual/5/installation/requirements)

**Review by docrozza — APPROVED** [review]

**Pull request merged**